### DISCUSSION TITLE
github.io - Wrong magic number for PMTiles archive

### DISCUSSION THREAD

**Discussion by kbatchu** [protomaps/PMTiles#283] [Q&A] [answered]
Hello,
I created a public repository on github.io and uploaded my pmtiles file. However, When I try to load it through my OpenLayers app using this [url] (https://kbatchu.github.io/acs5yr2021popdotdensity500highresv2.pmtiles), I get the 'Wrong magic number for PMTiles archive' error in the browser console. When I load this same file from my local drive, it loads fine. So I know that the file is ok, which I had created using tippecanoe.
![pmtiles_code](https://github.com/protomaps/PMTiles/assets/34347771/ad4f3fbf-dca0-42d3-b98d-59d1b0edf4f1).
I also loaded this url through the web version of postman and verified that it returns a valid response, although it returns the entire file. Any ideas why this is happening?
- Thanks,
- Kiran

**Comment by bdon** [comment]
When I ask for https://kbatchu.github.io/acs5yr2021popdotdensity500highresv2.pmtiles I get a 133 byte text response instead of the pmtiles archive. maybe something in GitHub pages configuration?

**Comment by kbatchu** [comment]
I am new to github.io pages. I currently do not have any html pages. I just have a couple of pmtiles files that I uploaded using Git LFS. I have this setting turned on for my repository. What am I missing?
![image](https://github.com/protomaps/PMTiles/assets/34347771/bb4bd528-5d40-4ffa-8a1b-9d2361088782)

**Reply by bdon** [reply]
I don't think GitHub pages works with Git LFS

**Reply by kbatchu** [reply]
I understand that Git LFS is just a command line tool to upload files > 100MB to GitHub repository. Would Git LFS change the file structure in some way when uploading?

**Reply by bdon** [reply, marked as answer]
It doesn't change the file structure, it does affect the storage strategy inside GitHub which likely means it cannot be served like a normal file through GitHub pages

**Reply by kbatchu** [reply]
You are right. I found a similar discussion here: https://github.com/git-lfs/git-lfs/issues/1342. Thanks.
